data/governments.txt

```diff
@@ -566,14 +567,21 @@ government "Hai Merchant (Human)"
 	"friendly hail" "friendly civilian"
 	"hostile hail" "hostile civilian"
 
-color "governments: Hai (Unfettered)" .55 .27 .76
+government "Hai (Governor Escort)"
+	"display name" "Hai"
+	swizzle 0
+	bribe .02
+	"friendly hail" "friendly hai"
+	"friendly disabled hail" "friendly disabled hai"
+	"hostile hail" "hostile hai"
+	"hostile disabled hail" "hostile disabled hai"
+  color "governments: Hai (Unfettered)" .55 .27 .76
```

Comment:
```suggestion
	color "governments: Hai (Unfettered)" .55 .27 .76
```

Reply:
it's not supposed to be linked to that government; it's for all UHai govs